opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipEditionOverview.tsx

```diff
@@ -275,6 +275,18 @@ Omit<StixCoreRelationshipEditionOverviewProps, 'queryRef'>
     stixCoreRelationshipValidator,
   );
 
+  // necessary for stop_time because the validator includes a reference to another value (start_time)
```

Comment:
If the validator refers to the wrong value, should it be fixed?

Reply:
it's not a wrong value, you need it to compare start_time and stop_time